types/express-ws/index.d.ts

```diff
@@ -31,10 +31,10 @@ declare namespace expressWs {
     interface Instance {
         app: Application;
         applyTo(target: RouterLike): void;
-        getWss(): ws.Server;
+        getWss(): ws.WebSocketServer;
     }
 
-    type WebsocketRequestHandler = (ws: ws, req: express.Request, next: express.NextFunction) => void;
+    type WebsocketRequestHandler = (ws: ws.WebSocket, req: express.Request, next: express.NextFunction) => void;
```

Comment:
`ws` is a synonym of `ws.WebSocket`, although I think the new code is easier to understand.